pandas/tests/io/test_sql.py

```diff
@@ -253,9 +253,13 @@ def _get_exec(self):
         else:
             return self.conn.cursor()
 
-    def _load_iris_data(self, datapath):
+    @pytest.fixture(params=[('io', 'data', 'iris.csv')])
+    def _load_iris_data(self, datapath, request):
```

Comment:
Given that it's now a fixture, let's remove the initial underscore in the function name.

Reply:
Sure @gfyoung that makes sense! Will go ahead and do this, any other feedback from anyone?

Reply:
Updated